src/main/java/com/microsoft/lst_bench/common/TaskExecutor.java

```diff
@@ -80,10 +85,18 @@ private EventInfo writeFileEvent(Instant startTime, String id, Status status) {
     return eventInfo;
   }
 
-  private EventInfo writeStatementEvent(Instant startTime, String id, Status status) {
-    EventInfo eventInfo =
-        ImmutableEventInfo.of(
-            experimentStartTime, startTime, Instant.now(), id, EventType.EXEC_STATEMENT, status);
+  private EventInfo writeStatementEvent(
+      Instant startTime, String id, Status status, String payload) {
+    EventInfo eventInfo = null;
+    if (payload != null) {
+      ImmutableEventInfo.of(
+              experimentStartTime, startTime, Instant.now(), id, EventType.EXEC_EXPERIMENT, status)
```

Comment:
Shouldn't this be `EventType.EXEC_STATEMENT`?

Reply:
Good catch, thanks!